bigtop-bigpetstore/bigpetstore-transaction-queue/Dockerfile

```diff
@@ -13,21 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 FROM centos:7
-MAINTAINER jay@apache.org
+LABEL maintainer="jay@apache.org"
 RUN yum update -y
-RUN yum install -y java-1.8.0-openjdk unzip wget
-RUN yum install -y java-1.8.0-openjdk-devel
+RUN yum install -y java-1.8.0-openjdk
 
 WORKDIR /opt/
 
-# Get Bigtop
-# This comes with a gradlew wrapper we can use.
-RUN wget http://www.apache.org/dist/bigtop/bigtop-1.0.0/bigtop-1.0.0-project.tar.gz
-RUN  tar -xvf bigtop-1.0.0-project.tar.gz
+COPY build/libs/bigpetstore-transaction-queue-all-1.0.jar /opt
```

Comment:
Sorry I'm not familiar with BPS.
So the idea here is to build bigpetstore-transaction-queue-all-1.0.jar locally and committed in the docker image?

Reply:
Yeah.  The old link was broken and I figured anyone wanting to build the dockerfile themself is probably capable of building the JAR too.